### PR TITLE
Merge 4.8.1 into 4.8.2

### PR DESCRIPTION
**Pull request by damarisg**
|Related issue|
|:--:|
| #5050 |

## Description
Merge 4.8.1 into 4.8.2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
+- Add AWS Custom Buckets Integration tests ([#4675](https://github.com/wazuh/wazuh-qa/pull/4675)) \- (Framework + Tests)
```
Comment (jnasselle) [inline CHANGELOG.md:22]: Duplicated in line 32

File: CHANGELOG.md [hunk 2]
```diff
@@ -50,13 +58,32 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+- Fix initial scans tests ([5032](https://github.com/wazuh/wazuh-qa/pull/5032)) \- (Framework + Tests)
+- Handle VDT data missing in wazuh-db API ([5014](https://github.com/wazuh/wazuh-qa/pull/5014)) \- (Framework + Tests)
```
Comment (jnasselle) [inline CHANGELOG.md:62]: PR number don't follow hashtag semantics

File: CHANGELOG.md [hunk 3]
```diff
@@ -50,13 +58,32 @@ All notable changes to this project will be documented in this file.
...
+- Fixed x-axis labels in data-visualizer script ([#4987 ](https://github.com/wazuh/wazuh-qa/pull/4987)) \- (Framework) 
+- Fix monitoring module for e2e tests ([#4959](https://github.com/wazuh/wazuh-qa/pull/4959)) \- (Framework)
```
Comment (jnasselle) [inline CHANGELOG.md:64]: Dupe from line 69

File: CHANGELOG.md [hunk 4]
```diff
@@ -32,6 +33,13 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
+- Reduced test_shutdown_message runtime ([#4986](https://github.com/wazuh/wazuh-qa/pull/4986)) \- (Tests) 
+- Change e2e vd configuration keystore ([#4952](https://github.com/wazuh/wazuh-qa/pull/4952)) \- (Framework)  
+- Updating tests after removing references to the legacy vulnerability detector module ([#4872](https://github.com/wazuh/wazuh-qa/pull/4872)) \- (Tests)  
+- Fix test cluster performance ([#4780](https://github.com/wazuh/wazuh-qa/pull/4780)) \- (Tests)
```
Comment (jnasselle) [inline CHANGELOG.md:39]: Duplicated from line 80

**Review by jnasselle — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (damarisg) [reply CHANGELOG.md]: Solved https://github.com/wazuh/wazuh-qa/pull/5063/commits/772857d06a2dea4feaeba7aed697da896e071667

File: CHANGELOG.md (hunk 3 above)
Comment (damarisg) [reply CHANGELOG.md]: Solved https://github.com/wazuh/wazuh-qa/pull/5063/commits/772857d06a2dea4feaeba7aed697da896e071667

File: CHANGELOG.md (hunk 2 above)
Comment (damarisg) [reply CHANGELOG.md]: Solved https://github.com/wazuh/wazuh-qa/pull/5063/commits/772857d06a2dea4feaeba7aed697da896e071667

File: CHANGELOG.md (hunk 1 above)
Comment (damarisg) [reply CHANGELOG.md]: Solved https://github.com/wazuh/wazuh-qa/pull/5063/commits/772857d06a2dea4feaeba7aed697da896e071667

File: CHANGELOG.md [hunk 5]
```diff
@@ -50,13 +55,32 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+- Fix initial scans tests ([#5032](https://github.com/wazuh/wazuh-qa/pull/5032)) \- (Framework + Tests)
+- Handle VDT data missing in wazuh-db API ([#5014](https://github.com/wazuh/wazuh-qa/pull/5014)) \- (Framework + Tests)
+- Fixed x-axis labels in data-visualizer script ([#4987](https://github.com/wazuh/wazuh-qa/pull/4987)) \- (Framework)
+- Fix monitoring module for e2e tests ([#4959](https://github.com/wazuh/wazuh-qa/pull/4959)) \- (Framework)
```
Comment (jnasselle) [inline CHANGELOG.md:61]: Dupe from line 66

**Review by jnasselle — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (rauldpm) [reply CHANGELOG.md:61]: Done in https://github.com/wazuh/wazuh-qa/pull/5063/commits/9c4fc1ab3d533bb1469a490401090672f142f6ac

**Review by jnasselle — APPROVED** [review]

**Pull request merged**